LayoutTests/fast/shadow-dom/imperative-slot-update-1.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<body>
+<div id="host"><div class="fail">FAIL</div><div class="pass">PASS</div></div>
+<style>
+#host > div { width: 100px; height: 100px; }
+.fail { background: red; }
+.pass { color: green; background: green; }
+</style>
+<script>
+
+if (window.testRunner)
+    testRunner.waitUntilDone();
+
+const shadowRoot = host.attachShadow({mode: 'closed', slotAssignment: 'manual'});
+const slot = document.createElement('slot');
+shadowRoot.appendChild(slot);
+slot.assign(document.querySelector('.fail'));
+
+requestAnimationFrame(() => {
```

Comment:
Why does there need to be any asynchrony in this test? We always do a rendering update at the end of the test.

Reply:
@smfr : oh, the point is to invalidate style after style had been updated.